index.d.ts

```diff
@@ -582,6 +584,30 @@ export declare interface User {
   [key: string]: string | undefined
 }
 
+export declare interface Appsec {
```

Comment:
Let's tag the methods as experimental/beta/unstable for now

Reply:
done, added
```
@beta This method is in beta and could change in the future versions
```
in each method description

Reply:
Made a small commit to fix the wording